src/jdk.jfr/share/classes/jdk/jfr/internal/consumer/RepositoryFiles.java

```diff
@@ -213,7 +212,7 @@ private boolean updatePaths() throws IOException {
                 pathSet.remove(time);
                 pathLookup.remove(remove);
             }
-            Collections.sort(added, (p1, p2) -> p1.compareTo(p2));
+            added.sort((p1, p2) -> p1.compareTo(p2));
```

Comment:
'added.sort(Path::compareTo)'

Reply:
Can't we just use natural ordering `null` here?

Reply:
Replaced with `Collections.sort` without comparator argument. I think it's a bit easier to read than with `null`.
https://github.com/openjdk/jdk/pull/5229#discussion_r695525255